minijinja/src/environment.rs

```diff
@@ -226,6 +226,32 @@ impl<'source> Environment<'source> {
         self._render_str("<string>", source, Value::from_serializable(&ctx))
     }
 
+    /// Parses and renders a template block from a string in one go.
+    ///
+    /// In some cases you really only need a template to be rendered once from
+    /// a string and returned.  The internal name of the template is `<string>`.
+    ///
+    /// ```
+    /// # use minijinja::{Environment, context};
+    /// let env = Environment::new();
+    /// let rv = env.render_block_str("{% block myblock %}Hello {{ name }}{% endblock %}", "myblock", context! { name => "World" });
+    /// println!("{}", rv.unwrap());
+    /// ```
+    ///
+    /// **Note on values:** The [`Value`] type implements `Serialize` and can be
+    /// efficiently passed to render.  It does not undergo actual serialization.
+    #[cfg(feature = "multi_template")]
+    pub fn render_block_str<S: Serialize>(
```

Comment:
I would probably not add this method. If you have blocks, you most likely have loaded templates anyways. If someone later wants that function it can still be added.